src/templates/du.conf.j2

```diff
@@ -158,4 +150,13 @@ log_config: {
     mac_log_level      = "info";
     rlc_log_level      = "info";
     f1ap_log_level     = "info";
-};
\ No newline at end of file
+};
+{% if simulation_mode %}
+rfsimulator: {
+    serveraddr = "server";
```

Comment:
Is this an address? Will this work?

Reply:
Not sure what that is exactly, but I've tested the workload with this config file and it worked.